apk_info/apk_info_test.go

```diff
@@ -0,0 +1,25 @@
+package apk_info
+
+import (
+	"path"
+	"runtime"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_GetAPKPackageName(t *testing.T) {
+	// Given
+	_, filename, _, ok := runtime.Caller(0)
+	require.True(t, ok)
+	mockAPKPath := path.Dir(filename) + "/testdata/app-debug-androidTest.apk"
```

Comment:
We have a separate repository storing large test files: https://github.com/bitrise-io/sample-artifacts.
Using it brings a bit of an overhead to the test implementation, but smaller step repo means a faster step execution.